settings.gradle

```diff
@@ -1,6 +1,7 @@
 pluginManagement {
   repositories {
     maven { url ossimMavenProxy }
+    mavenCentral()
```

Comment:
The only one you should need is the ossimMavenProxy. There is no mavenCentral on JWICS.

Reply:
Rgr.  Added that when i began upgrading things.  Seems to compile fine without.